charts/fluid/fluid/templates/webhook/webhook.yaml

```diff
@@ -60,6 +60,10 @@ spec:
           - name: FILE_PREFETCHER_IMAGE
             value: {{ include "fluid.controlplane.imageTransform" (list .Values.webhook.filePrefetcher.imagePrefix .Values.webhook.filePrefetcher.imageName .Values.webhook.filePrefetcher.imageTag . ) }}
           {{- end }}
```

Comment:
Should Helm chart check If the native-sidecar value Is valid  based on the Kubernetes version?